heartbeat/mysql

```diff
@@ -411,9 +411,9 @@ get_read_only() {
     local read_only_state
 
     read_only_state=`$MYSQL $MYSQL_OPTIONS_REPL \
-        -e "SHOW VARIABLES" | grep read_only | awk '{print $2}'`
+        -ss -e "SHOW GLOBAL VARIABLES LIKE 'read_only'"
 
-    if [ "$read_only_state" = "ON" ]; then
+    if [ $read_only_state =~ '*ON' ]; then
```

Comment:
According to http://shellcheck.net, this has two issues:
* SC2074 Can't use =~ in [ ]. Use [[..]] instead.
* Don't quote rhs of =~, it'll match literally rather than as a regex. [1]

[1]: https://github.com/koalaman/shellcheck/wiki/SC2076